tox.ini

```diff
@@ -74,3 +74,10 @@ commands =
 commands =
     rm -f .coverage
     py.test --cov=scalyr_agent/ --cov=tests/
+
+
+[testenv:rpm]
+deps =
```

Comment:
Just a heads up - if those tests will eventually rely on some environment variables being set, you will need to specify ``passenv`` option - https://tox.readthedocs.io/en/latest/config.html#conf-passenv

By default, tox tries to ensure a "clean" execution environment for each tox run, so it doesn't preserve most of the environment variables (kinda like sudo vs sudo -E).